pimd/pim_ifchannel.c

```diff
@@ -589,7 +599,7 @@ static int on_ifjoin_expiry_timer(struct thread *t)
 
   ch->t_ifjoin_expiry_timer = NULL;
 
-  ifjoin_to_noinfo(ch);
+  ifjoin_to_noinfo(ch, 1);
```

Comment:
Is it possible to change this to avoid using a raw number (1 in this case)? Just to improve the readability of the code without having to go back ifjoin_to_noinfo() to understand what 1 means?a true/false would at least makes it clear that this is a boolean value, but maybe you can do better.  :-)